components/select_team/select_team.jsx

```diff
@@ -80,6 +80,7 @@ export default class SelectTeam extends React.Component {
         }
     }
 
+    // Is there a reason we are doing this rather than just using `haveISystemPermission(me, CREATE_TEAM)`?
```

Comment:
@saturninoabril is there a reason we are doing all this complicated stuff here? To me it looks like we could just use `haveISystemPermission()` to do these checks, but it looked deliberately done differently so wondering if you remember why?

Reply:
It was a workaround before when permission scheme phase 1 was initially introduced.  You're right about `haveISystemPermission`.

Reply:
Awesome. I'll modify it to use just that then.